### PR TITLE
Alexey 134 web pages

### PR DESCRIPTION
**Pull request by rezniknn**

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -1,31 +1,63 @@
-<title>Easy Phi - test measurement equipment platform</title>
+<!DOCTYPE html>
```
Comment (user2589) [inline static/index.html:1]: Please fix w3c validator warnings

File: static/css/jquery-ui.css [hunk 2]
```diff
@@ -0,0 +1,1225 @@
+/*! jQuery UI - v1.11.4 - 2015-06-23
```
Comment (user2589) [inline static/css/jquery-ui.css]: Please use minimized version instead

File: static/js/main.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    //Set click handlers for Clear and Export buttons
+    $("#btn_clear").on('click', function(){
+        $('#log_console').empty();
+    });
+
+    $("#btn_export").on('click', function(){
+        //TBD
+    });
+})
+function generateHeaderContainer(id, name, used_by) {
```
Comment (user2589) [inline static/js/main.js]: function in global space.. use namespaces

File: static/js/main.js [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+    beforeActivate: function(event, ui) {
+
+         // The accordion believes a panel is being opened
+        if (ui.newHeader[0]) {
+            var currHeader  = ui.newHeader;
+            var currContent = currHeader.next('.ui-accordion-content');
+         // The accordion believes a panel is being closed
+        } else {
+            var currHeader  = ui.oldHeader;
+            var currContent = currHeader.next('.ui-accordion-content');
```
Comment (user2589) [inline static/js/main.js]: what's the difference to L121?

File: static/js/main.js [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+        }
+
+        if (currContent.hasClass('disabled')) {
+            return false;
+        }
+         // Since we've changed the default behavior, this detects the actual status
+        var isPanelSelected = currHeader.attr('aria-selected') == 'true';
+
+         // Toggle the panel's header
+        currHeader.toggleClass('ui-corner-all',isPanelSelected).toggleClass('accordion-header-active ui-state-active ui-corner-top',!isPanelSelected).attr('aria-selected',((!isPanelSelected).toString()));
```
Comment (user2589) [inline static/js/main.js]: should be refactored

File: static/js/main.js [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+
+        // Toggle the panel's icon
+        currHeader.children('.ui-icon').toggleClass('ui-icon-triangle-1-e',isPanelSelected).toggleClass('ui-icon-triangle-1-s',!isPanelSelected);
```
Comment (user2589) [inline static/js/main.js]: same here

File: static/js/main.js [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+
+         // Toggle the panel's content
+        currContent.toggleClass('accordion-content-active',!isPanelSelected)
```
Comment (user2589) [inline static/js/main.js]: use of magic values

File: static/js/main.js [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+        $('#log_console').empty();
+    });
+
+    $("#btn_export").on('click', function(){
+        //TBD
+    });
+})
+function generateHeaderContainer(id, name, used_by) {
+    //display slot index
+    var result = "<h3 class = 'module_header' id='module_header" + id + "'>" + id + ". ";
```
Comment (user2589) [inline static/js/main.js]: run through jslint

File: static/js/main.js [hunk 9]
```diff
@@ -0,0 +1,187 @@
+$(document).ready(function (){
+    //Initialise Accordion widget
+    initAccordion();
+
+    //get Platform info
+    $.get("/api/v1/info", function(data){
+	document.getElementById("txt_wlcm_msg").innerHTML = data['welcome_message'];
```
Comment (user2589) [inline static/js/main.js]: why don't you use jquery methods here?

File: static/js/main.js [hunk 10]
```diff
@@ -0,0 +1,187 @@
...
+    $('#send_scpi' + id).on('click', function() {
+        //Send SCPI command to the module
+        var command = $('#scpi_command' + id).val();
+        writeToConsole("Sending command to module #" + id + ": " + command + "\n");
+        $.post("/api/v1/module?slot=" + id, command, function(data){
+            writeToConsole("Response from module #" + id + ":\n" + data + "\n");
+        });
+    });
+}
+function initAccordion(){
```
Comment (user2589) [inline static/js/main.js]: it's used only in one place, why not to make it inline?

File: static/css/style.css [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+#modules_list_container {
+    margin:auto;
+    width:80%;
+    height: 100%;
+    padding: 5px;
+}
+
+footer {
+    clear:both;
+    position: absolute;
```
Comment (user2589) [inline static/css/style.css:86]: it is either clear:both or position:absolute

File: static/css/style.css [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+    margin-top: 5px;
+    height: 80%;
+    width: 100%;
+    overflow: auto;
+    border: 1px solid black;
+    resize:none;
+}
+
+#modules_list_container {
+    margin:auto;
```
Comment (user2589) [inline static/css/style.css:78]: missing min-width

**Comment by user2589** [comment]
Not that I like it, but we'll take care of it later. Ship it

**Comment by mansuleman** [comment]
Shipt it

**Pull request merged**